testSuite/scripts/test_upload_block_blob.py

```diff
@@ -835,3 +835,26 @@ def test_follow_symlinks_upload(self):
 
         self.assertEquals(x.TransfersCompleted, "10")
         self.assertEquals(x.TransfersFailed, "0")
+
+    def test_overwrite_posix_properties(self):
```

Comment:
❔ I don't really see why this needed the python testing framework-- We could modify the relevant file properties in beforeJobRun & create a differently sized file on the destination to overwrite the properties on; then we could add a hook for beforeValidation to correct our test files to ensure that we did not overwrite the file, but instead just the properties.

Reply:
Put this in the testSuite in cmd

Reply:
Not entirely what I meant either. I meant using the `e2etest` suite.